content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md

```diff
@@ -17,7 +17,7 @@ cluster if necessary.
 ## kubeadm upgrade guidance
 
 Every upgrade process might be a bit different, so we've documented each minor upgrade process individually.
-Please check these documents out for more detailed how-to-upgrade guidance:
+For more detailed how-to-upgrade guidance for each version, see the following resources:
```

Comment:
Suggest rewrite: "For more version-specific upgrade guidance, see the following resources:"

Reply:
@Bradamant3 did that change.

Reply:
@neolit123 a better editor than I is shepherding the whole 1.11 release. If you wouldn't mind changing it, their version is better. misty and I are meeting soon to make sure we don't inflict colliding edits on any more files for you. My apologies!

Reply:
ok. :+1: